lighthouse-cli/bin.ts

```diff
@@ -229,9 +235,16 @@ function lighthouseRun(addresses: Array<string>, config: Object, flags: Object)
   return lighthouse(address, flags, config)
     .then((results: Results) => Printer.write(results, outputMode, outputPath))
     .then((results: Results) => {
-      if (outputMode === Printer.OutputMode[Printer.OutputMode.pretty]) {
+      if (flags.view) {
+        const filename = `${assetSaver.getFilenamePrefix({url: address})}.report.html`;
+        const fileAbsPath = `${__dirname}/server/reports/${filename}`;
+        Printer.write(results, 'html', fileAbsPath);
+        opn(`http://localhost:${_REPORT_SERVER_PORT}/reports/${filename}`);
+
```

Comment:
It does not completely override pretty.
Pretty prints result to console then saves report.html to CWD. I think we probably don't need to save the html to CWD if we already saved it to "server/reports" and opened it in a browser.